runelite-client/src/main/java/net/runelite/client/plugins/emojis/EmojiPlugin.java

```diff
@@ -63,12 +82,77 @@
 	@Inject
 	private ChatMessageManager chatMessageManager;
 
+	@Inject
+	private Provider<MenuManager> menuManager;
+
+	@Inject
+	private ClientToolbar clientToolbar;
+
+	@Inject
+	private EmojiConfig config;
+
+	@Inject
+	private ClientThread clientThread;
+
+	static final String CONFIG_GROUP_KEY = "emoji";
+	private static final String EMIT = "Show";
+	private static final String MENU_TARGET = "Emoji Key";
+	private static final WidgetMenuOption FIXED_INVENTORY_ALL_TAB = new WidgetMenuOption(EMIT,
+		MENU_TARGET, WidgetInfo.FIXED_VIEWPORT_ALL_CHAT_TAB);
+	private static final WidgetMenuOption RESIZABLE_INVENTORY_ALL_TAB = new WidgetMenuOption(EMIT,
+		MENU_TARGET, WidgetInfo.RESIZABLE_VIEWPORT_ALL_CHAT_TAB);
+	private static final int EMOJI_PER_LINE = 7;
+	//	 Holds current emoji message, if any
+	private EmojiEmitMessage currentMessage = null;
 	private int modIconsStart = -1;
 
+	@Provides
+	EmojiConfig provideConfig(ConfigManager configManager)
+	{
+		return configManager.getConfig(EmojiConfig.class);
+	}
+
+	/**
+	 * Remove tags, except for &lt;lt&gt; and &lt;gt&gt;
+	 *
+	 * @return String with tags removed
+	 */
+	private static String removeTags(String str)
```

Comment:
This method has still been moved compared to where it was originally. This should be appearing on what is currently L333 of this file. Look at the diff view of this PR to see what I mean. (down where you've added `timeoutEmojiMessages()`)

Reply:
Yeah, sorry, the diff was confusing me. _Best as I can tell_ this is put back now.